src/game/QuizMaster.ts

```diff
@@ -12,10 +13,20 @@ export class QuizMaster {
 
     private _quiz: Quiz;
 
-    get quiz() : Quiz {
+    get quiz(): Quiz {
         return this._quiz
     }
-    
+
+    game() {
```

Comment:
stulle? What happens when i call game? Does the game start, does it end? Start over? New game?

Reply:
does 'quizGameFlow' work?